variants/ancientmediterranean/ancientmediterranean.go

```diff
@@ -22,19 +22,18 @@ const (
 var Nations = []dip.Nation{Rome, Greece, Egypt, Persia, Carthage}
 
 var AncientMediterraneanVariant = common.Variant{
-	Name:        "Ancient Mediterranean",
-	Graph:       func() dip.Graph { return AncientMediterraneanGraph() },
-	Start:       AncientMediterraneanStart,
-	Blank:       AncientMediterraneanBlank,
-	Phase:       classical.Phase,
-	ParseOrders: orders.ParseAll,
-	ParseOrder:  orders.Parse,
-	OrderTypes:  orders.OrderTypes(),
-	Nations:     Nations,
-	PhaseTypes:  cla.PhaseTypes,
-	Seasons:     cla.Seasons,
-	UnitTypes:   cla.UnitTypes,
-	SoloWinner:  common.SCCountWinner(18),
+	Name:       "Ancient Mediterranean",
+	Graph:      func() dip.Graph { return AncientMediterraneanGraph() },
+	Start:      AncientMediterraneanStart,
+	Blank:      AncientMediterraneanBlank,
+	Phase:      classical.Phase,
+	Parser:     orders.ClassicalParser,
+	OrderTypes: orders.ClassicalParser.OrderTypes(),
```

Comment:
From zond: "The OrderTypes field is redundant now, it could just be replaced with Parser.OrderTypes() wherever it's used."

Reply:
@zond The only reason I kept it was for the REST API interface. OrderTypes is exposed as part of a variant, although I'm not sure it's used at all.

Reply:
Ah, but the REST variants use a separate struct with extra fields anyway, IIRC. Might put the extracted order types there?

Reply:
No - I just tested this. Currently the interface from this file (common.Variant) contains the JSON help strings, and is exposed over the REST API.

I think for now let's leave this as is, and maybe in the future we could extract a second interface.

Unless there's some way to set a default value for OrderTypes in the variant struct using the variant's Parser?

Reply:
If you are talking about the /variants URLs, it's actually https://github.com/zond/diplicity/blob/master/variants/variants.go#L70 (which contains an anonymous common.Variant) that's exported over REST.

Into this RenderVariant type isnthe graph and start phases copied at https://github.com/zond/diplicity/blob/master/variants/variants.go#L102.

My idea was that you could move the OrderTypes field from common.Variant to variants.RenderVariant.

Would that work?